proc_GDS.py

```diff
@@ -1,18 +0,0 @@
-from pyspecdata import *
-fl = figlist_var()
-for date,id_string,PW in [
-        ('190201','SpinCore_pulses','0.88'),
-        ]:
-    filename = date+'_'+id_string+'.h5'
-    id_string = id_string.replace('MWE_','Example ') # for plotting purposes
-    nodename = 'accumulated_'+date
-    s = nddata_hdf5(filename+'/'+nodename,
-            directory = getDATADIR(
-                exp_type = 'test_equip'))
-    s.set_units('t','s')
-    s = s['ch',0].C
-    print(ndshape(s))
-    fl.next('idk')
-    for i,x in enumerate(s.getaxis('capture')):
-        fl.plot(s['capture',i],alpha=0.2)
-    fl.show();quit()
```

Comment:
we have replaced this file with specific processing of things captured on the gds for example proc_triwave, proc_capture_nutation, proc_chirp. This is a general just plot what I captured. Not sure how specific/general you want these scripts to be... Could possibly be made into a second level function if those other scripts do the exact same thing?

Reply:
if you have the files that do what you want, for GDS, probably OK to delete this